src/services/queue-service.js

```diff
@@ -53,6 +55,13 @@ function setQueueAPIVersion(response) {
   }
 }
 
+function setQueueUseSSE(response) {
+  if (response && response.headers) {
+    var useSSE = response.headers["x-cio-use-sse"];
+    useSSE && useSSE.toLowerCase() === "true" ? settings.setUseSSEFlag(true) : settings.setUseSSEFlag(false);
+  }
+}
+
```

Comment:
👀 Readability nitpick: We could avoid nesting & simplify this function with optional chaining

```suggestion
function setQueueUseSSE(response) {
  const useSSE = response?.headers?.["x-cio-use-sse"]?.toLowerCase() === "true";
  settings.setUseSSEFlag(useSSE);
}
```